tools/appimage/make-appimage.sh

```diff
@@ -0,0 +1,79 @@
+#! /bin/bash
```

Comment:
there should be no space after !

Reply:
That's really a question of personal preference. I prefer this because of the increased readability. If you insist, I'll change it.

For the record, this script passes shellcheck without any issues.

Reply:
You don't actually read the shebang, do you?
I know that extra space is optional, but it's so unusual, and all other scripts in this repo do not have it, so I guess the norm here is to have no optional space.